### PR TITLE
chore(DTFS2-7535-7538): gov design library - word break, role attributes

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
- GOV design library has been updated.
- This PR updates all UI microservices to adhere to the latest "work break" and "role attributes" patterns/requirements.

## Resolution :heavy_check_mark:
- Remove unnecessary styles of `word-break: break-word;`, replace with `govuk-text-break-word` class or SCSS mixin.
- Remove some unnecessary `role` attributes to some certain elements - these were in place to support legacy browsers which are no longer supported by GOV design, as per the [5.3.0 release notes](https://github.com/alphagov/govuk-frontend/releases/tag/v5.3.0).
- Add E2E tests for asserting there is a `nav` element in pagination. Note: It does not seem possible to achieve with cheerio component tests.

### CODE REVIEW

File: gef-ui/styles/styles.scss [hunk 1]
```diff
@@ -101,7 +100,7 @@ $govuk-new-link-styles: true;
         margin-top: 0;
       }
       white-space: pre-line;
-      word-break: break-word;
+      @include govuk-text-break-word();
```
Comment (Zainzzkk) [inline gef-ui/styles/styles.scss:103]: does this need an empty line before it? codacy comment

File: trade-finance-manager-ui/templates/_macros/pagination.njk [hunk 2]
```diff
@@ -5,11 +5,11 @@
   {% set paginationRoute = params.paginationRoute %}
   {% set queryString = params.queryString %}
 
-  <div class="pagination" role="navigation">
+  <div class="pagination" data-cy="pagination">
 
     {% if totalPages > 1 %}
 
-      <div data-cy="pagination" role="navigation">
+      <nav>
```
Comment (Zainzzkk) [inline trade-finance-manager-ui/templates/_macros/pagination.njk:12]: should we give the nav a data-cy?

File: gef-ui/styles/styles.scss (hunk 1 above)
Comment (ttbarnes) [reply gef-ui/styles/styles.scss:103]: I'm not seeing any Codacy comments?

File: trade-finance-manager-ui/templates/_macros/pagination.njk (hunk 2 above)
Comment (ttbarnes) [reply trade-finance-manager-ui/templates/_macros/pagination.njk:12]: I don't think so, this was actually semantically incorrect - the parent div (which now has `data-cy`) is the actual wrapping pagination div.

File: gef-ui/styles/styles.scss (hunk 1 above)
Comment (Zainzzkk) [reply gef-ui/styles/styles.scss:103]: weird, there was just one 30mins ago which has disappeared

Comment (Zainzzkk) [reply gef-ui/styles/styles.scss:103]: image

Comment (ttbarnes) [reply gef-ui/styles/styles.scss:103]: Not sure, maybe something to do with previous merge conflicts.

**Review by Zainzzkk — APPROVED** [review]

**Review by Zainzzkk — COMMENTED** [review]
do all the integrities need to be updated?

**Comment by ttbarnes** [comment]
> do all the integrities need to be updated?

No, only `mojFilterHide` JS has been updated here.

**Pull request merged**